package.json

```diff
@@ -12,9 +12,12 @@
   ],
   "license": "MIT",
   "main": "./build/angular-gravatar.js",
-  "dependencies": {},
+  "dependencies": {
+    "angular": "^1.2.5"
```

Comment:
Does npm accept this kind of version ranges: `>=1.0.0 <2.0.0`? This is what's used in [bower.json](https://github.com/wallin/angular-gravatar/blob/master/bower.json#L24).